### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mdhaber**
- You had written 6mm x 2.5mm magnet, but the link has 2mm thick or 3mm thick magnets. Which should we use?
- You had written 11mm diameter tack, but I had purchased 8mm/9mm tacks. Does that work with the CAD models?
- What material should these be printed from?

Also, would you check the [rules](https://docs.google.com/document/d/1JPMAodRuwTGvhz5UEUWVJHI-WIk5GcbBl3c8BVmr7d0/edit) for consistency between what is written about the leaves (e.g. heights and radii) against the models?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +25,17 @@ Example Assembly: https://a360.co/3RiKY86
...
+# Painting
+All surfaces are spray painted with colors as follows:
+
+- Stems - [Satin Moss Green](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Satin-Moss-Green-General-Purpose-Spray-Paint-334075/307244920)
+- Healthy leaves - [Gloss Meadow Green](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Gloss-Green-General-Purpose-Spray-Paint-334039/307244868)
+- Unhealthy leaves - [Satin Green Apple](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Satin-Green-Apple-General-Purpose-Spray-Paint-334070/307244897)
+- Flowers - [Flat White](https://www.homedepot.com/p/Rust-Oleum-Painter-s-Touch-2X-12-oz-Flat-White-General-Purpose-Spray-Paint-334021/307244842)
 
+# Assembly Instructions
+1. Print an appropriate number of stalks, leaves, flowers, and stems (of various lengths) per the rules. If desired, CAD files may be modified as needed to achieve reliable press fits in the steps below (given your printer / material).
```
Comment (mdhaber) [inline README.md:37]: Could you read the rules and suggest an appropriate number of each based on your understanding? (I'll check you.)

File: README.md [hunk 2]
```diff
@@ -1,23 +1,18 @@
 # ASABE 2024 Robotics Competition Plant
-This repository contains the files needed to print (and possibly modify for tolerancing) the simulation plant for the 2024 ASABE robotics competition.
+This repository contains the files needed to print the simulation plant for the 2024 ASABE robotics competition.
 
 # Terminology
-Trunk - The center core which fits into the game board and holds stems
+Stalk - The center core which fits into the game board and holds stems
+Stem - The part that connects the trunk and leaves/flowers. It holds a 6mmx2.5mm magnet (press fit).
```
Comment (mdhaber) [inline README.md:6]:
```suggestion
Stem - The part that connects the stalk and leaves/flowers. It holds a 6mm x 2 mm magnet (press fit).

```

**Comment by mdhaber** [comment]
@DominicChm could you merge this?

**Comment by DominicChm** [comment]
Closing this in favor of #2 which should have these changes along with more detailed instructions and updated models

**Pull request closed without merging**